meerk40t/gui/choicepropertypanel.py

```diff
@@ -27,8 +27,27 @@ def __init__(self, *args, context: Context = None, choices=None, **kwds):
             choices = self.context.lookup("choices", choices)
             if choices is None:
                 return
-        self.choices = choices
+        # Let's see whether we have a section and a page property...
+        for c in choices:
+            try:
+                dummy = c["section"]
+            except KeyError:
+                c["section"] = ""
+            try:
+                dummy = c["page"]
+            except KeyError:
+                c["page"] = ""
+        # print ("Choices: " , choices)
+        self.choices = sorted(sorted(choices, key=lambda d: d["section"]), key=lambda d: d["page"])
```

Comment:
So alphabetical? Wouldn't this sort into different orders if it was translated?

Reply:
Changed